src/components/Table.css

```diff
@@ -0,0 +1,54 @@
+table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+td {
+  padding: 5px;
+}
+
+.tableDetails {
+  display: flex;
+  flex-direction: column;
+}
+
+.externalLink {
+  display: block;
+}
+
+.tableHeaders {
+  font-size: 1.2em;
+  height: 50px;
+  border: 2px solid black;
+}
+
+.tableThumbnail img {
+  width: 50px;
+  height: 50px;
+}
+
+.tableListing:nth-child(odd) {
+  background-color: rgb(244, 243, 243);
+}
+
+.tableListing:nth-child(even) {
+  background-color: gainsboro;
```

Comment:
This color name is a new one to me! Cool!

Reply:
oh yeah it's my go-to gray :)